layouts/shortcodes/pages/compatibility-products.html

```diff
@@ -8,47 +8,120 @@
+        <div class="match-height-item-by-row padding-20 flex-column vertical-align">
+          {{ if .image }}
+            <img width="{{ .image_width }}" class="img img-responsive" src="{{ .image }}" alt="{{ .name }} logo">
+          {{ else }}
+            <div class="icon-backdrop">
+              <div class="ratio"></div>
+              <div class="icon-container"><i class="fa fa-download"></i></div>
             </div>
-            <div class="flex-grow text-center flex-column">
-              <p class="margin-bottom-0"><strong>{{ .name }}</strong></p>
-              <p class="flex-grow margin-bottom-10">{{ .vendor }}</p>
-              <p>
-              {{ range .versions }} 
-                <a class="compatibility-link" href="{{ .compatibility }}">{{ .version }}</a>
-              {{ end }}           
-              </p>
-            </div>
-          </div>
+          {{ end }}
         </div>
+      </div>
       {{ end }}
-      <div class="clearfix"/>
     </div>
   </div>
+
+{{ range $.Site.Data.compatible_products.sets }}
+  <div role="tabpanel" class="tab-pane tab-pane" id="tab-{{ .jakartaee_version }}">
+    {{ range .items }}
+      {{ $title:=.title }}
+      <div class="product-list">
+        <h2 class="h3 margin-top-0">{{ .title }}</h2>
+        {{/*  <h2 class="text-center heading-line" id="{{ .title | urlize }}"><span style="background-color:#fff;">{{ .title }}</span></h2>  */}}
+        <div class="row">
+         {{ if isset . "items" }}
+         {{ range sort .items "name" "asc"}}
+           {{/*  {{ range .items }}  */}}
+            <div class="col-xs-24 col-sm-12 col-lg-6 padding-bottom-20">
+              <div class="match-height-item-by-row padding-20 flex-column">
+                <div class="center-block padding-left-30 padding-right-30 padding-bottom-10 padding-top-10 grid-center" data-mh="{{ $title | urlize }}">
+                  {{ if .image }}
+                  <a href="{{ .download }}"><img width="{{ .image_width }}" class="img img-responsive" src="{{ .image }}" alt="{{ .vendor }}'s {{ .name }} logo"></a>
+                  {{ else }}
+                  <div class="icon-backdrop">
+                    <div class="ratio"></div>
+                    <div class="icon-container"><i class="fa fa-download"></i></div>
+                  </div>
+                  {{ end }}
+                </div>
+                <div class="flex-grow text-center flex-column">
+                  <p class="margin-bottom-0"><strong>{{ .name }}</strong></p>
+                  <p class="flex-grow margin-bottom-10">{{ .vendor }}</p>
+                  <p>
+                  {{ range .versions }} 
+                    <a class="compatibility-link" href="{{ .compatibility }}">{{ .version }}</a>
+                  {{ end }}           
+                  </p>
+                </div>
+              </div>
+            </div>
+         {{ end }}
+         {{ else }}
+           coming soon
```

Comment:
We should put this into a tag, and also create an i18n label for it

Reply:
Sounds good. M not sure before we merge this will they provide me any data, if they're going to publish with "coming soon" or any words, I will add i18n label for it and put into a tag. 

Thx for reminding!